lib/engine/game/g_1880/step/simple_draft.rb

```diff
@@ -39,10 +39,16 @@ def process_bid(action)
 
             @log << "#{player.name} chooses #{minor.full_name} (#{minor.name})"
 
+            assign_bcr_share_to_fi(action.entity, minor) if player.shares.find { |s| s.corporation.id == 'BCR' }
+
             @round.next_entity_index!
             action_finalized
           end
 
+          def assign_bcr_share_to_fi(player, fi)
+            @game.assign_share_to_fi(player.shares.first.corporation, fi)
+          end
```

Comment:
this logic is needed as BCR is floated before the FI is chosen. As soon as the owner of the BCR picks a FI, a share of the BCR goes to the FI.